lib/rmt/downloader.rb

```diff
@@ -0,0 +1,137 @@
+module RMT
```

Comment:
Yeah, that's left over since before `lib/rmt.rb` came to life.  I'll clean that up.